app/src/main/java/com/whereuat/whereu/ContactRetriever.java

```diff
@@ -0,0 +1,69 @@
+package com.whereuat.whereu;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.database.Cursor;
+import android.net.Uri;
+import android.provider.ContactsContract;
+import android.content.Context;
+
+/**
+ * Created by whites5 on 3/22/16.
+ */
+public class ContactRetriever {
+    private static Context mContext;
+    private static String phone_num;
+    private String contact_id;
+    private String contact_name;
+    private static final String[] projection = {
```

Comment:
private static final members (ie. constants) should be upper case underscore so `UPPERCASE_UNDERSCORE` or `PROJECTION`